packages/compass-editor/src/json-editor.tsx

```diff
@@ -570,12 +683,18 @@ const BaseEditor: React.FunctionComponent<EditorProps> & {
           ...historyKeymap,
           ...foldKeymap,
           ...completionKeymap,
-          // Breaks keyboard navigation out of the editor, but we want that
-          // https://codemirror.net/examples/tab/
-          indentWithTab,
+          breakFocusOutBinding,
+          {
+            key: 'Tab',
+            run(view) {
+              console.log(view);
```

Comment:
```suggestion
```

Reply:
Ah, this whole thing actually shouldn't be there, commited by accident

Reply:
Removed